File: executable/julia/RdBu-heatmap.jl
using Plotly
Plotly.signin("test-runner", "9h29fe3l0x")

size = 50
z = rand(size, size)
for r = 1:size
  for c = 1:size
        z(r,c) = sqrt(r*c/size^2)

data = [
  [
    "z" => z, 
    "scl" => "RdBu", 
    "type" => "heatmap"
  ]
]
layout = ["title" => "RdBu"]

response = Plotly.plot([data], ["layout" => layout, "filename" => "RdBu-heatmap", "fileopt" => "overwrite"])
plot_url = response["url"]

File: auto-docs/executables/nodejs/RdBu_heatmap.js
var plotly = require('plotly')('TestBot', 'r1neazxo9w')

var unpack = require('ndarray-unpack');
var zeros = require('zeros');
var fill = require('ndarray-fill');

var size = 50;
var z = unpack(fill(zeros([size, size]), function(c,r) {
  return r+c;
}));

var data = [
  {
    z: z, 
    scl: "RdBu", 
    type: "heatmap"
  }
];
var layout = {title: "RdBu"};

var graph_options = {filename: "RdBu-heatmap", fileopt: "overwrite", layout: layout, auto_open: "false"}
plotly.plot(data, graph_options, function (err, msg) {
    console.log(msg);
});